tests/test_certificates.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2020 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+
+import pytest
+import sgtk
+
+from tank_test.tank_test_base import setUpModule, interactive
+from base_test import TestDesktopServerFramework
+
+skip_on_ci = pytest.mark.skipif("CI" in os.environ, reason="These tests require manual intervention to execute.")
+
+class TestCertificates(TestDesktopServerFramework):
+
+    @skip_on_ci
+    def test_certificate_creation_flow(self):
+        """
+        Ensure certificates registration and creation work.
+        """
+
+        from sgtk.platform.qt import QtGui
+
+        if QtGui.QApplication.instance() is None:
+            self.app = QtGui.QApplication([])
+
+        QtGui.QMessageBox.warning(None, "", "You will be prompted to accept updates to the keychain during this test.")
```

Comment:
Display a message to the user to warn them that they will be prompted for their credentials/fingersprint/click on ok by the OS, to avoid developers getting confused by pop-ups during the tests.